source/UIAHandler/remote.py

```diff
@@ -0,0 +1,27 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2021-2022 NV Access Limited, Zahari Yurukov, Babbage B.V., Joseph Lee

```

Comment:
This list of contributors is incorrect - probably it was mistakenly copied from some other file
```suggestion
# Copyright (C) 2021-2022 NV Access Limited
```